packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -705,34 +636,30 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         }
     }
 
-    private async readCommit(id: string): Promise<IOdspSnapshotCommit | null> {
+    private async readCommit(id: string): Promise<api.ISnapshotTree | null> {
```

Comment:
Similar feedback with naming, I think we expect to get snapshot ID, and name of function should not mention commit?

Reply:
Changed it to readTree.